EasyPost/Utilities/Internal/Extensions/Dictionaries.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+
+namespace EasyPost.Utilities
+{
+    public static class Dictionaries
+    {
+        /// <summary>
+        ///     Converts a <see cref="Dictionary{TKey,TValue}"/> of string, object? (nullable) key-value pairs to a dictionary of string, object key-value pairs
+        ///     by omitting key-value pairs with null values.
+        /// </summary>
+        /// <param name="dictionary">A <see cref="Dictionary{TKey,TValue}"/> to convert.</param>
+        /// <returns>A <see cref="Dictionary{TKey,TValue}"/> of string, object pairs.</returns>
+        public static Dictionary<string, object> ConvertToStringNonNullableObjectDictionary(Dictionary<string, object?> dictionary)
```

Comment:
### Author's Note

This was ported from the Extensions library. It's needed because our functions expect `Dictionary<string, object>`-type (non-nullable) dictionaries, but the parameters serialization creates `Dictionary<string, object?>`-type (nullable) dictionaries. This function converts a nullable dictionary to a non-nullable dictionary by simply removing all key-pairs with null values.